ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor.hs

```diff
@@ -549,16 +556,36 @@ peerSelectionGovernorLoop tracer
       -- get the current time after the governor returned from the blocking
       -- 'evalGuardedDecisions' call.
       now <- getMonotonicTime
+
       let Decision { decisionTrace, decisionJobs, decisionState } =
             timedDecision now
-          !newCounters = peerStateToCounters decisionState
+
+          !outboundConnectionsState =
+            if activePeers st
+               `Set.isSubsetOf`
+               LocalRootPeers.keysSet (localRootPeers st)
```

Comment:
We should use `decisionState` not `st`.

* `st` is the initial state of this loop state
* `decisionState` is the state after executing a peer selection action

Reply:
Fixed.